docs/markdown/Creating-releases.md

```diff
@@ -31,6 +31,14 @@ Meson then takes this archive and tests that it works by doing a full
 `compile` + `test` + `install` cycle. If all these pass, Meson will
 then create a `SHA-256` checksum file next to the archive.
 
+## Modifying the dist directory before creating the archive
+
+Modification to the checked out files like generating files or
+setting version info can be done by dist scripts.
+Make a script run during `meson dist` with this:
```

Comment:
Maybe in combination with the above suggestion, remove "this"?

Reply:
Or actually maybe remove the line entirely.

    Modification to the checked out files like generating files or
    setting version info can be done with dist scripts:
    ```meson
    [[#meson.add_dist_script]]('distscript.sh')
    ```